content/en/docs/concepts/storage/volumes.md

```diff
@@ -1391,72 +963,489 @@ plugin path on each node and in some cases the control plane nodes as well.
 Pods interact with FlexVolume drivers through the `flexvolume` in-tree volume plugin.
 For more details, see the [FlexVolume](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-storage/flexvolume.md) examples.
 
-## Mount propagation
+### persistentVolumeClaim {#persistentvolumeclaim}
 
-Mount propagation allows for sharing volumes mounted by a container to
-other containers in the same pod, or even to other pods on the same node.
+A `persistentVolumeClaim` volume is used to mount a
+[PersistentVolume](/docs/concepts/storage/persistent-volumes/) into a pod. PersistentVolumeClaims are a way for users to "claim" durable storage (such as a GCE PersistentDisk or an iSCSI volume) without knowing the details of the particular cloud environment.
 
-Mount propagation of a volume is controlled by the `mountPropagation` field
-in `Container.volumeMounts`. Its values are:
+See the information about [PersistentVolumes](/docs/concepts/storage/persistent-volumes/) for more
+details.
 
-* `None` - This volume mount will not receive any subsequent mounts
-  that are mounted to this volume or any of its subdirectories by the host.
-  In similar fashion, no mounts created by the container will be visible on
-  the host. This is the default mode.
+## External Volume Providers
 
-  This mode is equal to `private` mount propagation as described in the
-  [Linux kernel documentation](https://www.kernel.org/doc/Documentation/filesystems/sharedsubtree.txt)
+### awsElasticBlockStore {#awselasticblockstore}
 
-* `HostToContainer` - This volume mount will receive all subsequent mounts
-  that are mounted to this volume or any of its subdirectories.
+An `awsElasticBlockStore` volume mounts an Amazon Web Services (AWS) [EBS
+Volume](https://aws.amazon.com/ebs/) into your pod.  Unlike
+`emptyDir`, which is erased when a pod is removed, the contents of an EBS
+volume are preserved and the volume is merely unmounted.  This means that an
+EBS volume can be pre-populated with data, and that data can be shared
+between pods.
 
-  In other words, if the host mounts anything inside the volume mount, the
-  container will see it mounted there.
+{{< caution >}}
+You must create an EBS volume using `aws ec2 create-volume` or the AWS API before you can use it.
```

Comment:
BTW, Kubernetes now offers [dynamic provisioning](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/) for storage.

Reply:
noted, thanks!